ops/bin/_env.sh

```diff
@@ -9,4 +9,6 @@ set -a
 source $DOTENV_PATH
 set +a
 
-
+export PGPASSWORD=$POSTGRES_PASSWORD
+export PGUSER=$POSTGRES_USER
+export PGHOST=$POSTGRES_HOST
```

Comment:
Noticed some missing newlines at the end of files... maybe missing that editor config?